### PR TITLE
Update for Flarum beta 15

### PR DESCRIPTION
**Pull request by jslirola**

### CODE REVIEW

File: extend.php [hunk 1]
```diff
@@ -27,10 +30,37 @@
...
+        })
+        ->serializeToForum('jslirola.login2seeplus.image', 'jslirola.login2seeplus.image', function ($val) {
+            return $val;
+        })
+        ->serializeToForum('jslirola.login2seeplus.php', 'jslirola.login2seeplus.php', function ($val) {
+            return $val;
+        })
+        ->serializeToForum('jslirola.login2seeplus.code', 'jslirola.login2seeplus.code', function ($val) {
+            return $val;
+        }),
```
Comment (askvortsov1) [inline extend.php:50]: You don't need 
```
function ($val) {
    }	            return $val;
```

you can just leave that 3rd argument blank

File: src/HideContentInPostPreviews.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace JSLirola\Login2SeePlus;
+
+class HideContentInPostPreviews extends FormatContent
+{
+    public function getContent($serializer, $attributes)
```
Comment (askvortsov1) [inline src/HideContentInPostPreviews.php:18]: Rename this to `__invoke`,  and add $model as a middle argument, and return attributes at the end (or split that into 2 methods), then in the extender, you can just do: 
`->mutate(HideContentInPostPreviews)`

**Review by askvortsov1 — CHANGES_REQUESTED** [review]

**Review by askvortsov1 — APPROVED** [review]

File: src/HideContentInPostPreviews.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        if (!$serializer->getActor()->isGuest())
+            return $attributes;
+
+        $s_summary_links = $this->settings->get('jslirola.login2seeplus.link', false);
+
+        if (!is_null($newHTML) && $s_summary_links == 1)
+            $newHtml = preg_replace('/(<a((?!PostMention).)*?>)[^<]*<\/a>/is',
+                '[' . $this->get_link('jslirola-login2seeplus.forum.link') . ']', $newHtml);
+
+        return $attributes;
```
Comment (askvortsov1) [inline src/HideContentInPostPreviews.php:31]: I think you're missing `$attributes['contentHtml'] = $newHTML;`

**Review by askvortsov1 — CHANGES_REQUESTED** [review]

File: extend.php (hunk 1 above)
Comment (jslirola) [reply extend.php]: Done!  :)

File: src/HideContentInPostPreviews.php (hunk 3 above)
Comment (jslirola) [reply src/HideContentInPostPreviews.php:33]: Yes, you're right. Thank you.

**Pull request merged**